Minio.Functional.Tests/MintLogger.cs

```diff
@@ -77,10 +77,37 @@ public MintLogger(string testName,string function, string description,TestStatus
         this.args = args;
         this.status = status.AsText();
       }  
-      public void Log() {
+      public void Log()
+      {
+            var sb = new StringBuilder();
+
+            sb.Append("function="); sb.AppendLine(this.function);
+            sb.Append("duration="); sb.AppendLine(this.duration.ToString() );
+            sb.Append("name=");     sb.AppendLine(this.name     );
+            sb.Append("alert=");    sb.AppendLine(this.alert    );
+            sb.Append("message=");  sb.AppendLine(this.message  );
+            sb.Append("error=");    sb.AppendLine(this.error    );
+            sb.Append("status=");   sb.AppendLine(this.status   );
+            sb.Append("args=");
+
+            if (this.args == null)
+            {
+                sb.AppendLine("[]");
+            }
+            else
+            {
+                foreach (var kv in this.args)
+                {
+                    sb.Append("[");
+                    sb.Append(kv.Key);
+                    sb.Append("]=");
+
+                    sb.AppendLine(kv.Value);
+                }
+            }
+
 
-          Console.Out.WriteLine(JsonConvert.SerializeObject(this,Formatting.None,
-            new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore }));
```

Comment:
@SanSYS, since Minio.FunctionalTests.csproj is retaining the NewtonSoft.JSON dependency, you can leave the Log() function code as it was, i.e. we just want lines 82 and 83 that you removed. The functional tests output needs to be in json format.
```
{"name":"minio-dotnet: BucketExists_Test","function":"Task<bool> BucketExistsAsync(string bucketName, CancellationToken cancellationToken = default(CancellationToken))","args":{"bucketName":"miniodotnetkxu3g"},"duration":2477,"status":"PASS"}
```

Reply:
Ok, thanks!